client/galaxy/scripts/mvc/tools.js

```diff
@@ -388,7 +389,7 @@ var ToolSearch = Backbone.Model.extend({
         this.timer = setTimeout(function () {
             // log the search to analytics
             ga( 'send', 'pageview', galaxy_config.root + '?q=' + q );
-            $.get(self.attributes.search_url, { query: q }, function (data) {
+            $.get( tool_search_url, { q: q }, function ( data ) {
```

Comment:
Well I would usually set the Backbone model urlRoot and then fetch it from a view but given there is no view and the binding is done on the model level I just put the url in the model itself. It makes more sense to me.